phase2/outline
# fix: install playwright + its dependencies

fix #166 

Testing Steps:
* Open this PR in Gitpod
* After everything finished loading, open a new terminal
* Run `yarn test`
* Confirm everything ran flawlessly.

<a href="https://gitpod.io/#https://github.com/phase2/outline/pull/187"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
